stock-tracker/src/features/chart/component/chartButton.tsx

```diff
@@ -1,9 +1,10 @@
 import React, { FC } from "react";
+import { ChartTimes } from "../redux/actions";
 
 type ChartButtonProps = {
   current: string;
```

Comment:
Shouldn't this be `current: ChartTimes`?

Reply:
correct, changed now.